packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ffi';
```

Comment:
why did you import dart:ffi?

Reply:
I did not, it appears VS imported it for reasons I am not sure of. Removed